src/sentry/release_health/duplex.py

```diff
@@ -1265,7 +1283,7 @@ def get_num_sessions_per_project(
         rollup: Optional[int] = None,  # rollup in seconds
     ) -> Sequence[ProjectWithCount]:
         schema = [(ComparatorType.Exact, ComparatorType.Counter)]
-        should_compare = lambda _: _coerce_utc(start) > self.metrics_start
+        should_compare = lambda: _coerce_utc(start) > self.metrics_start
```

Comment:
In `get_project_sessions_count`, we should probably change this: https://github.com/getsentry/sentry/blob/0d01e00a711490455fccb6bcd83d9fabcdd5c8db/src/sentry/release_health/duplex.py#L1238-L1243

to this:
```python
 # We verified the correctness of the metrics implementation manually. 
 # The results still differ because the sessions impl gets its results 
 # from hourly aggregations. 
schema = ComparatorType.Ignore 
  
should_compare = lambda: _coerce_utc(start) > self.metrics_start 
```

Reply:
ah right, because otherwise we don't return the result